doc/man/man3/crypto_chacha20_encrypt.3monocypher

```diff
@@ -83,7 +83,9 @@ in bytes.
 .It Fa stream
 The raw Chacha20 stream.
 .It Fa stream_size
-The size of the stream, in bytes.
+The size of
+.Fa stream ,
+in bytes.
```

Comment:
I personally vote for "length" and no "the". So,

    Length of
    .Fa stream ,
    in bytes.

Though I care more about consistency than this particular phrasing.